dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAIntrospection.java

```diff
@@ -26,30 +26,116 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
 import com.ibm.ws.ffdc.FFDCFilter;
 import com.ibm.ws.jpa.diagnostics.JPAORMDiagnostics;
 import com.ibm.ws.jpa.diagnostics.class_scanner.ano.jaxb.classinfo10.ClassInfoType;
-import com.ibm.ws.jpa.diagnostics.puscanner.PersistenceUnitScannerResults2;
+import com.ibm.ws.jpa.diagnostics.puscanner.PersistenceUnitScannerResults;
 
 /**
  *
  */
 public class JPAIntrospection {
+    private static final TraceComponent tc = Tr.register(JPAORMDiagnostics.class,
```

Comment:
This is the wrong class. Should be JPAIntrospection.class

Reply:
Ah!  Great catch, thanks!